### PR TITLE
Update caption

### PR DESCRIPTION
**Pull request by BethanyShen-ISD**
## Test comment

### CODE REVIEW

File: _slides/03-BethanyShen-ISD.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Updated caption.
```
Comment (nbisd3287) [inline _slides/03-BethanyShen-ISD.md:6]: I have been here and it looks good

Comment (nbisd3287) [inline _slides/03-BethanyShen-ISD.md:6]:
```suggestion
** Updated caption **
```

**Review by nbisd3287 — APPROVED** [review]
Good to deploy 😄

**Comment by BethanyShen-ISD** [comment]
Merging code change.

**Pull request merged**